Header header
float64 exposure_us
float64 aperture
float64 gain_db
float64 wb_red
float64 wb_green
float64 wb_blue
float64 estimated_illuminance_lux
